lib/foods.js

```diff
@@ -101,7 +104,7 @@ function Food(name, calories) {
 function displayFoods() {
   foods = setFoods();
   foods.forEach(function(foodItem) {
-    prependRow(foodItem.name, foodItem.calories);
+    prependRow('#food-list',foodItem.name, foodItem.calories);
```

Comment:
i wonder if we could possibly pass in "this" to wherever we're calling this function ("displayFoods") so that we're not hardcoding this and it's more flexible. just a suggestion!